import_ldraw.py

```diff
@@ -928,6 +964,41 @@ def getColorValue(line, value):
     if value in line:
         n = line.index(value)
         return line[n + 1]
+        
+def linkedParts():
+    """Clean-up design by linking identical parts (mesh/color)"""
+    colors = []
+    for color in bpy.data.materials:
+        colors.append(color.name)
```

Comment:
This should be rewritten using list comprehension: `colors = [color.name for color in bpy.data.materials]`